lib/ecto/schema.ex

```diff
@@ -346,12 +346,12 @@ defmodule Ecto.Schema do
 
     ## Source
 
-    The `:source` field is a tuple tracking the database source (table or
-    collection) where the struct is or should be persisted. It is represented
-    as a tuple consisting of two fields:
+    The `:source` tracks the (table or collection) where the struct is or should
+    be persisted to.
 
-      * source - tracking the table or collection name.
-      * prefix - tracking the ecto prefix of the table or collection.
+    ## Prefix
+
+    Tracks the struct prefix in the data storage.
```

Comment:
Would "Tracks the source's prefix in the data storage" be more appropriate?

Reply:
:+1: